src/main/scala/rocket/Rocket.scala

```diff
@@ -642,7 +642,7 @@ class Rocket(implicit p: Parameters) extends CoreModule()(p)
   csr.io.counters foreach { c => c.inc := RegNext(perfEvents.evaluate(c.eventSel)) }
 
   if (enableCommitLog) {
-    val pc = Wire(SInt(width=xLen))
+    val pc = Wire(UInt(width=xLen))
     pc := wb_reg_pc
```

Comment:
I think it's because the width of wb_reg_pc isn't known at that point.  ```getWidth``` would ideally print a more helpful error message...

Let's try this instead:

    val pc = Wire(SInt(width=xLen))
    pc := wb_reg_pc.asSInt

:)